jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/SlideGroupAssert.java

```diff
@@ -34,4 +37,17 @@ public SlideGroupAssert visibleSlidesPosition(int position) {
         jdiAssert(element().slidesPosition(), Matchers.is(String.format("transform: translateX(%spx);", position)));
         return this;
     }
+
+    @JDIAction("Assert that '{name}' has center active slide")
+    public SlideGroupAssert centerActiveSlide() {
+        int lstSliderSize = element().getAllSlide().size();
```

Comment:
I see you use only list count. You can change getAllSlide() to getSlidesCount() in current situation